### ISSUE TITLE
Import / Export

### ISSUE THREAD

**Issue by Giorgi625** [JoomCoder-com/JoomCCK#227]
Can you change the way how import export feature works? It was really tricky to export content from Cobalt and then import it to another website with Cobalt. 
I had to use this feature couple times and had a lot of struggle.

**Comment by andiamine** [comment]
JoomCCK doesn't have any export feature, i think you can export only true Packers, but Packers features still needs a lot of improvements and refactoring.

**Comment by Giorgi625** [comment]
Yes there is export feature. which is done using Formatter plugins. To export articles you need to enable one of the formatter plugin depending on what format file you want to get on export INI, JSON, CSV or XML. and then add some text to url of article list page, but trick is that it only will export field values which are displayed in list view template so if you do not show all fields then you need to add new template where all fields are shown and enable it for a moment for export and then get back to default template. This is why its tricky. also sometime exported files still needed some manual editing in some cases. 
At least this is how it worked on Cobalt according to Sergey who taught me to do so. 
As I see Mintjoomla again is not accessible so I can not find that post, on https://web.archive.org I was able to view its cashed version but there search does not work so it will be hard to find that topic.

**Comment by Giorgi625** [comment]
I just noticed in latest versions Formatter plugins were removed. So yes now only Packer is only export option

**Comment by andiamine** [comment]
Formatters already included in extended version. but we still didn't test them.
